commands/unequip.js

```diff
@@ -39,7 +39,7 @@ exports.run = (client, message, args) => {
 	{
 		case "TRINKET":
 		{
-			if(trinket == "NONE")
+			if(trinket == "NONE" || armor.length <= 0)
```

Comment:
wouldn't this want to check trinket.length?

Reply:
Yes, thank you! One of my players reported that bug but I still hadn't found it. Will resolve shortly.